apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -473,8 +474,8 @@ struct ChatView: View {
             VStack(alignment: .trailing, spacing: 4) {
                 ChatItemView(chatInfo: chat.chatInfo, chatItem: ci, showMember: showMember, maxWidth: maxWidth, scrollProxy: scrollProxy, revealed: $revealed, allowMenu: $allowMenu, audioPlayer: $audioPlayer, playbackState: $playbackState, playbackTime: $playbackTime)
                     .uiKitContextMenu(menu: uiMenu, allowMenu: $allowMenu)
-                if ci.reactions.count > 0 {
-                    chatItemReactions(ci.reactions)
+                if ci.content.msgContent != nil && ci.meta.itemDeleted == nil && ci.reactions.count > 0 {
```

Comment:
for received message - if the message is short and reactions row is wide, message bubble will be pushed to center

Reply:
if it's narrower - should it then be left-aligned on received?

Reply:
I think yes